README.md

```diff
@@ -3,27 +3,37 @@
 ## Due Date
 Due by 9am on August 12th
 
+## Wireframes
+For reference we have created an invision prototype to show you the flow of the react application which can be found here: https://bridgeschool.invisionapp.com/public/share/5GWTMSJMH
+
+Please note that you do not need to style the application to match this prototype, it is purely to show you the flow of the application. Feel free to style the application as much (or as little) as you'd like.
+
 ## Requirements
 
-- Create a landing page where a user should be able to input a valid git username and submit it.
+- Create a search form that allows you enter a github user name and submit it
 
-- After submitting the username, two lists should be displayed: 
+- After submitting the form, you should make an API call to fetch the inputted username's information
```

Comment:
Is the intention that they actually do an HTML submit, or more of a button onClick? If it's the latter it may help to not say "submit".

Reply:
I think for the scope of this we can just use a button onClick. Perhaps this is better wording?

```suggestion
- After clicking the search button is clicked, you should make an API call to fetch the inputted username's information
```

Reply:
Agreeable